doc/content/reference/email-templates/overriding.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Overriding Templates"
+description: ""
+weight: 6
+---
+
+> Note: We recommend getting familliar with the [html/template](https://golang.org/pkg/html/template/) template format first.
+
+## Template components
+
+An email template override has 3 components:
+
+1. The subject file, which contains the subject of the email and is named `<identifier>.subject.txt`. 
+2. The text contents file, which contains the contents of the email in text format and is named `<identifier>.txt`.
+3. The HTML contents file, which contains the contents of the email in HTML format and is named `<identifier>.html`.
+
+
+## Creating the overrides
+
+In order to override a template, one must provide all 3 files as part of the email templates repository and then provide them to a [fetcher]({{< ref "/concepts/fetchers.md" >}}).
+
+Let's consider that we want to override the email that a user receives once he registers, the validation email (`validate`). We need to create the following files:
```

Comment:
No gender in users, check all occurrences.

Always they/their/them

Reply:
Also what is `validate` here? Is there a list of identifiers? How does that work?

Reply:
Reiterated the explanation for `validate`.